packages/components/src/internal/components/samples/SampleDetailEditing.tsx

```diff
@@ -137,20 +166,48 @@ export class SampleDetailEditing extends PureComponent<Props, State> {
             />
         ) : null;
 
+        const parentDetailHeader = showRootSampleName ? (
+            <table className="table table-responsive table-condensed detail-component--table__fixed sample-aliquots-details-meta-table">
```

Comment:
Looks like there is another usage of this similar type of `<table>` in the `SampleAliquotDetailHeader.tsx`, would it be worth factoring this out to a little component to share between then (mainly just so that we don't have to keep these long set of class names in sync between the two). This might not be worth it, so just a suggestion.